apollo-router/Cargo.toml

```diff
@@ -44,11 +44,6 @@ dhat-ad-hoc = ["dhat"]
 # alerted early when something is wrong instead of receiving an invalid result.
 failfast = []
 
-# Enables usage of tokio-console with the router
-# tokio-console also requires at build time the environment variable
-# RUSTFLAGS="--cfg tokio_unstable"
-console = ["tokio/tracing", "console-subscriber"]
```

Comment:
This was added in https://github.com/apollographql/router/pull/1632 but hasn’t worked since https://github.com/apollographql/router/pull/2358 which removed relevant code in `apollo-router/src/plugins/telemetry/mod.rs`